lms/templates/shoppingcart/shopping_cart.html

```diff
@@ -9,140 +9,143 @@
-      <p id="expiry-msg">${_('{course_names} has been removed because the enrollment period has closed.').format(course_names=appended_expired_course_names)}</p>
-     % endif:
- <%
-    discount_applied = False
-    order_type = 'personal'
- %>
-
-  <section class="wrapper confirm-enrollment shopping-cart cart-view">
-    % for item, course in shoppingcart_items:
-       % if loop.index > 0 :
-        <hr>
-      %endif
-      % if item.order.order_type == 'business':
-       <% order_type = 'business' %>
-      %endif
-     <div class="user-data">
-      <div class="clearfix">
-      <div class="image">
-
-           <img style="width: 100%; height: 100%;" src="${course_image_url(course)}"
-                alt="${course.display_number_with_default | h} ${get_course_about_section(course, 'title')} Cover Image" />
-
-      </div>
-      <div class="data-input">
-        <h3>${_('Registration for:')} <span class="pull-right">${_('Course Dates:')}</span></h3>
-        <h1>${ course.display_name }</h1><span class="pull-right">${course.start_datetime_text()} -  ${course.end_datetime_text()}</span>
-        <hr />
-        <div class="three-col">
-            <div class="col-1">
-                % if item.list_price != None:
-                <% discount_applied = True %>
-                <div class="price">${_('Price per student:')} <span class="line-through">  ${currency_symbol}${"{0:0.2f}".format(item.list_price)}</span></div>
-                <div class="price green">${_('Discount Applied:')} <span>  ${currency_symbol}${"{0:0.2f}".format(item.unit_cost)} </span></div>
-                % else:
-                <div class="price">${_('Price per student:')} <span>  ${currency_symbol}${"{0:0.2f}".format(item.unit_cost)}</span></div>
-                % endif
-            </div>
-            <div class="col-2">
-              <div class="numbers-row">
-                <label for="students">${_('Students:')}</label>
-                <div class="counter">
-                    <input maxlength="3" title="Input qty and press enter." max="999" type="text" name="students" value="${item.qty}" id="${item.id}" data-unit-cost="${item.unit_cost}" data-qty="${item.qty}">
+        </div>
+        <div class="data-input">
+          <h3>${_('Registration for:')} <span class="pull-right">${_('Course Dates:')}</span></h3>
+          <h1>${ course.display_name }</h1><span class="pull-right">${course.start_datetime_text()} -  ${course.end_datetime_text()}</span>
+          <hr>
+          <div class="three-col">
+              <div class="col-1">
+                  % if item.list_price != None:
+                    <% discount_applied = True %>
+                    <div class="price">${_('Price per student:')} <span class="line-through">  ${currency_symbol}${"{0:0.2f}".format(item.list_price)}</span></div>
+                    <div class="price green">${_('Discount Applied:')} <span>  ${currency_symbol}${"{0:0.2f}".format(item.unit_cost)} </span></div>
```

Comment:
NIT: would be great to actually use "price-discounted" instead of "green", but that's technically existing styling so it is fine to leave as is.